### PR TITLE
RUM: Collecting browser errors

### PR DESCRIPTION
**Pull request by kayayarai**
### What does this PR do?
- Adds a page to the RUM browser section for with info about collecting browser errors. 
- Moves info on manual error collection from "advanced config" page to the new page.

### Motivation
DOCS-1362

### Preview

https://docs-staging.datadoghq.com/kari/rum-collect-errors/real_user_monitoring/browser/collecting_browser_errors/

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by hdelaby — APPROVED** [review]

File: content/en/real_user_monitoring/browser/collecting_browser_errors.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+  - link: "/real_user_monitoring/dashboards/"
+    tag: "Documentation"
+    text: "RUM Dashboards"
+---
+
+
+Front-end errors are collected with Real User Monitoring (RUM). The error message and stack trace are included when available.
+
+## Error Origins
+Front-end errors are split in 4 different categories depending on their `error.origin`:
```
Comment (sarina-dd) [inline content/en/real_user_monitoring/browser/collecting_browser_errors.md:26]:
```suggestion
Front-end errors are split into four different categories depending on their `error.origin`:
```

File: content/en/real_user_monitoring/browser/collecting_browser_errors.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+
+- **network**: XHR or Fetch errors resulting from AJAX requests. Specific attributes to network errors can be found [in the documentation][1].
```
Comment (sarina-dd) [inline content/en/real_user_monitoring/browser/collecting_browser_errors.md:28]:
```suggestion
- **network**: XHR or Fetch errors resulting from AJAX requests. Specific attributes to network errors can be found [in the network errors documentation][1].
```

File: content/en/real_user_monitoring/browser/collecting_browser_errors.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+- **source**: Unhandled exceptions or unhandled promise rejections (source-code related).
+- **console**: `console.error()` API calls.
+- **custom**: Errors sent with the [RUM `addError` API][2] default to `custom`.
```
Comment (sarina-dd) [inline content/en/real_user_monitoring/browser/collecting_browser_errors.md:31]:
```suggestion
- **custom**: Errors sent with the [RUM `addError` API][2] that default to `custom`.
```

File: content/en/real_user_monitoring/browser/collecting_browser_errors.md [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+
+## Error attributes
```
Comment (sarina-dd) [inline content/en/real_user_monitoring/browser/collecting_browser_errors.md:33]:
```suggestion
## Error Attributes
```

File: content/en/real_user_monitoring/browser/collecting_browser_errors.md [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+| Attribute       | Type   | Description                                                       |
+|-----------------|--------|-------------------------------------------------------------------|
+| `error.source`  | string | Where the error originates from (for example, `console` or `network`).     |
+| `error.type`    | string | The error type (or error code in some cases).                   |
+| `error.message` | string | A concise, human-readable, one-line message explaining the event. |
+| `error.stack`   | string | The stack trace or complementary information about the error.     |
+
+### Network errors
+
+Network errors include information about failing HTTP requests. The following facets are also collected:
```
Comment (sarina-dd) [inline content/en/real_user_monitoring/browser/collecting_browser_errors.md:46]:
```suggestion
Network errors include information about failing HTTP requests. The following facets are collected:
```

File: content/en/real_user_monitoring/browser/collecting_browser_errors.md [hunk 6]
```diff
@@ -0,0 +1,175 @@
...
+
+### Source errors
+
+Source errors include code-level information about the error. More information about the different error types can be found in [the MDN documentation][5].
+
+| Attribute       | Type   | Description                                                       |
+|-----------------|--------|-------------------------------------------------------------------|
+| `error.type`    | string | The error type (or error code in some cases).                   |
+
+## Collect errors manually
```
Comment (sarina-dd) [inline content/en/real_user_monitoring/browser/collecting_browser_errors.md:69]:
```suggestion
## Collect Errors Manually
```

File: content/en/real_user_monitoring/browser/collecting_browser_errors.md [hunk 7]
```diff
@@ -0,0 +1,175 @@
...
+
+```javascript
+addError(
+    error: unknown,
+    context?: Context,
+    source: ErrorSource.CUSTOM | ErrorSource.NETWORK | ErrorSource.SOURCE = ErrorSource.CUSTOM
+);
+```
+
+**Note**: The [Error Tracking][6] feature processes errors sent with source set to `custom` or `source` and that contain a stacktrace.
```
Comment (sarina-dd) [inline content/en/real_user_monitoring/browser/collecting_browser_errors.md:81]:
```suggestion
**Note**: The [Error Tracking][6] feature processes errors sent with source set to `custom` or `source` and that contain a stack trace.
```

File: content/en/real_user_monitoring/browser/collecting_browser_errors.md [hunk 8]
```diff
@@ -0,0 +1,175 @@
...
+
+| Attribute       | Type   | Description                                                       |
+|-----------------|--------|-------------------------------------------------------------------|
+| `error.type`    | string | The error type (or error code in some cases).                   |
+
+## Collect errors manually
+
+Monitor handled exceptions, handled promise rejections and other errors not tracked automatically by the RUM SDK with the `addError()` API:
+
+```javascript
```
Comment (sarina-dd) [inline content/en/real_user_monitoring/browser/collecting_browser_errors.md:73]: https://github.com/DataDog/dd-docs-tools/wiki/Markdown#code-blocks - ref the new code block as it has the c+p option.

**Review by sarina-dd — COMMENTED** [review]
Docs review completed

**Review by sarina-dd — APPROVED** [review]

**Pull request merged**